packages/web-config-server/src/apiV1/utils/fetchIndicatorValues/checkAllDataElementsAreDhisIndicators.js

```diff
@@ -9,14 +9,13 @@
 export const checkAllDataElementsAreDhisIndicators = async (models, dataElementCodes) => {
   const dataElements = await models.dataSource.find({
     code: dataElementCodes,
-    type: 'dataElement',
   });
 
   for (const dataElementCode of dataElementCodes) {
     const dataElement = dataElements.find(d => d.code === dataElementCode);
     if (
       !dataElement.config.dhisId ||
-      !['Indicator', 'ProgramIndicator'].includes(dataElement.config.dhisDataType)
+      !['ProgramIndicator'].includes(dataElement.config.dhisDataType)
```

Comment:
Removing hacky code. Will still need to keep the hacky code for fetching `ProgramIndicator`. But it's already handled here https://github.com/beyondessential/tupaia/pull/2334